evm/utils.js

```diff
@@ -1052,6 +1056,15 @@ const verifyContractByName = (env, chain, name, contract, args, options = {}) =>
 
 const isConsensusChain = (chain) => chain.contracts.AxelarGateway?.connectionType !== 'amplifier';
 
+/**
+ * Determines if a chain is a Hyperliquid chain based on its name or axelarId
+ * @param {Object} chain - The chain object containing name and axelarId properties
+ * @returns {boolean} - Returns true if the chain is a Hyperliquid chain, false otherwise
+ */
```

Comment:
fluff comments, also outdated. let's remove it? The function name makes it pretty clear and easy to maintain